app/views/establishments/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<% if can? :manage, @group %>
+  <section class="selection-container">
+      <span>Most recent</span>
+      <span><%= link_to "Settings", edit_user_registration_path %></span>
+  </section>
+
+  <section class="user-container">
+    <div>
+      <strong><%= current_user.name %></strong>
+    </div>
+    <span class="red-price">
+      <strong><%= number_to_currency(@group.establishments.sum(:amount)) %></strong>
+    </span>
+  </section>
+
+  <section class="establishments-container">
+    <p style="color: green"><%= notice %></p>
+      
+    <div class="establishments">
+      <% @establishments.each do |establishment| %>
+        <% if can? :manage, establishment %>
+          <%= link_to(group_establishment_path(@group, establishment)) do %>
+            <%= render establishment %>
+          <% end %>
+        <% end %>
+      <% end %>
+    </div>
+    
+    <%= link_to "Back to groups", groups_path, class: 'back-button' %>
```

Comment:
Instead of using the `Back to groups` button down below, kindly consider using a left arrow at the top as per the provided design.
| Your design  | Porvided design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/218731545-79e1f0f6-d06f-4924-a02d-a32f0ab2fc43.png) | ![image](https://user-images.githubusercontent.com/33268927/218731807-aa1c7d92-c289-420a-bfca-59321c10ad6a.png) |

![image](https://user-images.githubusercontent.com/33268927/218732715-d520e1f8-21db-4f47-9dbd-24d200837201.png)